src/conda/install.sh

```diff
@@ -114,8 +114,13 @@ if ! conda --version &> /dev/null ; then
     
     find "${CONDA_DIR}" -type d -print0 | xargs -n 1 -0 chmod g+s
 
-    # Temporary due to https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23491
+    # Temporary fixes
```

Comment:
Can you help bump the patch version? thanks! 
https://github.com/devcontainers/features/blob/5fe97a2b1f5dfd85a354beef09f2c5e08d3152cd/src/conda/devcontainer-feature.json#L3

Reply:
Sure, bumped version in 4dd890c86d40ac4bd1caf40274a7e533bd3364e6